src/applications/personalization/dashboard/helpers.jsx

```diff
@@ -43,6 +42,8 @@ import pensionConfig from 'applications/pensions/config/form.js';
 import disability526Config from 'applications/disability-benefits/526EZ/config/form.js';
 import bddConfig from 'applications/disability-benefits/bdd/config/form.js';
 import hlrConfig from 'applications/disability-benefits/996/config/form';
+import mdotConfig from 'applications/disability-benefits/2346/config/form';
+import mdotManifest from 'applications/disability-benefits/2346/manifest.json';
```

Comment:
nit: can you move this up with the other manifests?

Reply:
done